ironfish-rust-nodejs/src/structs/unsigned.rs

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use napi::{
+    bindgen_prelude::{Buffer, Result},
+    JsBuffer,
+};
+use napi_derive::napi;
+
+use ironfish::transaction::unsigned::UnsignedTransaction as UnsignedTransactionRust;
+
+use crate::to_napi_err;
+
+#[napi]
+pub struct UnsignedTransaction {
+    transaction: UnsignedTransactionRust,
+}
```

Comment:
nit - we use `Native` elsewhere (`NativeProposedTransaction`, `NativeTransaction`), should we do the same here?

Reply:
We do use Native, but then we remap to `Transaction` via JS. 
```
#[napi(js_name = "Transaction")]
pub struct NativeTransaction {
    transaction: ProposedTransaction,
}
```
My guess is that this was originally done because whoever wrote it didn't think to use an alias for the import.
IMO this is better from searchability perspective, open to converting to other mechanism though

Reply:
Talked with Hugh, I renamed back to our original semantics, we can change all these if we want to with a future refactor.